views/includes/grid-blok.pug

```diff
@@ -10,8 +10,6 @@
   .grid-element
     .grid-edge
       .grid-title
-        h2 #{classNames[0]}
+        h2 #{classNames}
```

Comment:
Why's this? The classnames are already printed in the `.grid-inside`, perhaps you'd only want the columns indicator here for brevity.